[2019-06-05 14:36:30,458] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: weatherDag.get_weather 2019-06-04T00:00:00+00:00 [queued]>
[2019-06-05 14:36:30,464] {__init__.py:1139} INFO - Dependencies all met for <TaskInstance: weatherDag.get_weather 2019-06-04T00:00:00+00:00 [queued]>
[2019-06-05 14:36:30,464] {__init__.py:1353} INFO - 
--------------------------------------------------------------------------------
[2019-06-05 14:36:30,464] {__init__.py:1354} INFO - Starting attempt 4 of 6
[2019-06-05 14:36:30,464] {__init__.py:1355} INFO - 
--------------------------------------------------------------------------------
[2019-06-05 14:36:30,629] {__init__.py:1374} INFO - Executing <Task(BashOperator): get_weather> on 2019-06-04T00:00:00+00:00
[2019-06-05 14:36:30,630] {base_task_runner.py:119} INFO - Running: [u'airflow', u'run', 'weatherDag', 'get_weather', '2019-06-04T00:00:00+00:00', u'--job_id', '11', u'--raw', u'-sd', u'DAGS_FOLDER//home/micah/Desktop/Work/ETL/weatherDag.py', u'--cfg_path', '/tmp/tmpMHVup5']
[2019-06-05 14:36:30,876] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather /home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/configuration.py:590: DeprecationWarning: You have two airflow.cfg files: /home/micah/airflow/airflow.cfg and airflow_home/airflow.cfg. Airflow used to look at ~/airflow/airflow.cfg, even when AIRFLOW_HOME was set to a different value. Airflow will now only read airflow_home/airflow.cfg, and you should remove the other file
[2019-06-05 14:36:30,876] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather   category=DeprecationWarning,
[2019-06-05 14:36:31,340] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather [2019-06-05 14:36:31,340] {__init__.py:51} INFO - Using executor SequentialExecutor
[2019-06-05 14:36:31,668] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather [2019-06-05 14:36:31,668] {__init__.py:305} INFO - Filling up the DagBag from /home/micah/Desktop/Work/ETL/airflow_home/dags/home/micah/Desktop/Work/ETL/weatherDag.py
[2019-06-05 14:36:31,725] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather Traceback (most recent call last):
[2019-06-05 14:36:31,726] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/bin/airflow", line 32, in <module>
[2019-06-05 14:36:31,726] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather     args.func(args)
[2019-06-05 14:36:31,726] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
[2019-06-05 14:36:31,727] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather     return f(*args, **kwargs)
[2019-06-05 14:36:31,727] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/bin/cli.py", line 501, in run
[2019-06-05 14:36:31,728] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather     dag = get_dag(args)
[2019-06-05 14:36:31,728] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather   File "/home/micah/Desktop/Work/ETL/airflow_virtualenv/local/lib/python2.7/site-packages/airflow/bin/cli.py", line 145, in get_dag
[2019-06-05 14:36:31,728] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather     'parse.'.format(args.dag_id))
[2019-06-05 14:36:31,729] {base_task_runner.py:101} INFO - Job 11: Subtask get_weather airflow.exceptions.AirflowException: dag_id could not be found: weatherDag. Either the dag did not exist or it failed to parse.
[2019-06-05 14:36:35,453] {logging_mixin.py:95} INFO - [2019-06-05 14:36:35,453] {jobs.py:2562} INFO - Task exited with return code 1
